static/app/utils/analytics/replayAnalyticsEvents.tsx

```diff
@@ -46,6 +46,9 @@ export type ReplayEventParameters = {
      */
     scale_bucket: 0 | 10 | 20 | 30 | 40 | 50 | 60 | 70 | 80 | 90 | 100;
   };
+  'replay.search': {
+    search_keys: string[];
```

Comment:
to match with @scefali's comment below. should make TS happy

```suggestion
    search_keys: string;
```